BEProjectsApp/models.py

```diff
@@ -1,3 +1,81 @@
 from django.db import models
+from django.utils import timezone
 
-# Create your models here.
+
+DOMAIN_CHOICES = [
+    (1, ("none")),
```

Comment:
I guess we want one more field in this model that is `approved` a boolean field. We missed it in the schema design. Because we want to show only those projects which have been approved by a teacher.

Reply:
oh ha. I will add it tomorrow.